src/MeetingsSDKAdapter.js

```diff
@@ -0,0 +1,153 @@
+import {MeetingsAdapter} from '@webex/component-adapter-interfaces';
+import {Observable, fromEvent, concat} from 'rxjs';
+import {filter, finalize, map, publishReplay, refCount} from 'rxjs/operators';
+
+const LOCAL = 'local';
+const MEDIA_READY = 'media:ready';
+
+/**
+ * The `MeetingsSDKAdapter` is an implementation of the `MeetingsAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to create and join webex meetings.
+ *
+ * @class MeetingsSDKAdapter
+ * @extends {MeetingsAdapter}
+ */
+export default class MeetingsSDKAdapter extends MeetingsAdapter {
+  constructor(datasource) {
+    super(datasource);
+    this.meetingsObservables = {};
+  }
+
+  /**
+   * Register the SDK meeting plugin to the device
+   * and sync the meeting collection from the server.
+   */
+  async connect() {
+    await this.datasource.meetings.register();
+    await this.datasource.meetings.syncMeetings();
+  }
+
+  /**
+   * Unregister the sdk meeting plugin from the device.
+   */
+  async disconnect() {
+    await this.datasource.meetings.unregister();
+  }
+
+  /**
+   * Retrieve the local device media and add them with
+   * the given default media settings to the meeting.
+   *
+   * @param {string} meetingID  meeting ID to retrieve the SDK meeting object to add local media to
```

Comment:
```suggestion
   * @param {string} meetingID  The meeting ID to retrieve the SDK meeting object to add local media to.
```

Reply:
I would actually suggest to get rid of `meeting`
```suggestion
   * @param {string} meetingID  ID to retrieve the SDK meeting object to add local media to
```